src/components/cc-explore.vue

```diff
@@ -21,17 +21,7 @@
-                        ></path>
-                      </g>
-                    </svg>
+                    <img :src="logoPath" alt="Logo" />
```

Comment:
I just realized, since we can pass in any logo URL, we should probably rename this to `logoUrl`. That way, the name would be consistent with other properties like `donationUrl`. We would just need to add a note in our README about mixed HTTP and HTTPS content, or strip out the protocol, if provided, from the `logoUrl`.